src/cli/litra-toggle.ts

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env node
+
+import { findDevice, turnOff, turnOn } from "./../driver";
```

Comment:
There are quite a few issues where the code doesn't align with our ESLint rules - fortunately, you should be able to fix most or all of them automatically by running `npm run lint-and-fix` and then committing the changes.